grpc/core/pom.xml

```diff
@@ -20,69 +20,79 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
Is this just reformatted?

Reply:
Please use the original formatting, so we can see the changes

Reply:
This is a 2 spaces -> 4 spaces indentation fix.

Reply:
Ok